core/src/main/java/org/stellar/anchor/dto/sep38/QuoteResponse.java

```diff
@@ -0,0 +1,29 @@
+package org.stellar.anchor.dto.sep38;
+
+import com.google.gson.annotations.SerializedName;
+import java.time.LocalDateTime;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class QuoteResponse {
```

Comment:
Suggest to prefix with `Sep38QuoteResponse` if the name conflicts with CallbackAPI.